### ISSUE TITLE
Fix example's PutHandler

### ISSUE THREAD

**Issue by klimesf** [jSquirrel/adaptive-restful-api#6]
```
java.lang.IllegalStateException: Transaction already active
	org.hibernate.jpa.internal.TransactionImpl.begin(TransactionImpl.java:58)
	cz.cvut.fel.adaptiverestfulapi.data.persistence.PutHandler.update(PutHandler.java:47)
	cz.cvut.fel.adaptiverestfulapi.data.persistence.PutHandler.put(PutHandler.java:40)
	cz.cvut.fel.adaptiverestfulapi.data.PutHandler.handle(PutHandler.java:28)
	cz.cvut.fel.adaptiverestfulapi.data.Dispatcher.process(Dispatcher.java:39)
	cz.cvut.fel.adaptiverestfulapi.core.Filter.resign(Filter.java:57)
	cz.cvut.fel.adaptiverestfulapi.serialization.Resolver.process(Resolver.java:26)
	cz.cvut.fel.adaptiverestfulapi.core.Filter.resign(Filter.java:57)
	cz.cvut.fel.adaptiverestfulapi.caching.Cache.process(Cache.java:22)
	cz.cvut.fel.adaptiverestfulapi.core.Filter.resign(Filter.java:57)
	cz.cvut.fel.adaptiverestfulapi.security.Authorization.process(Authorization.java:16)
	cz.cvut.fel.adaptiverestfulapi.core.Filter.resign(Filter.java:57)
	cz.cvut.fel.adaptiverestfulapi.security.Authentication.process(Authentication.java:19)
	cz.cvut.fel.adaptiverestfulapi.servlet.FilteredServlet.service(FilteredServlet.java:37)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
```

**Comment by klimesf** [comment]
This is not a PutHandler issue. This happens when an unknown request fucks up the whole fucking API which then remains locked up in transaction.

**Comment by klimesf** [comment]
Should be closed by 0ae9acf
